pkg/source/registry_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package source_test
+
+import (
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/source"
+)
+
+func TestGetRegistryWithProgress(t *testing.T) {
```

Comment:
not sure what value this is adding, feel free to delete this test file

Reply:
deleted